site/src/lib/playground.js

```diff
@@ -3,6 +3,7 @@ function createPlayground() {
     'use strict';
 
     var scriptTarget,
+        root,
```

Comment:
These changes seem pretty brittle,  I wonder if there's a better way? Maybe we could reference the file from the YAML front-matter and have the template spit out a relative path?